tests/solutions_for_students/test_block_world_problem.py

```diff
@@ -0,0 +1,27 @@
+from problems.blocks_world.blocks_world_problem import BlocksWorldProblem
+from problems.blocks_world.blocks_world_state import BlocksWorldState
+from problems.blocks_world.blocks_world_action import BlocksWorldAction
+
+
+def assert_equal(expected, got):
+    assert expected == got, f"expected to receive: {expected}, got: {got}"
+
+
+def test_there_are_no_actions_for_empty_state():
```

Comment:
`test_actions_return_no_actions_for_empty_state`

Hand't seen before test names with articles.
Rather they are constructed in the following manner:

```python
test_[FUNCTION_NAME]_[DESCRIPTION]
```